services/insights.ubuntu.com.yaml

```diff
@@ -4,6 +4,8 @@ kind: Service
 apiVersion: v1
 metadata:
   name: insights-ubuntu-com
+  labels:
+    role: proxy
```

Comment:
If we keep with the `role` label in a more generic use, this makes sense.
If we don't, we probably don't need to apply a `use_proxy` label to the service, only on the deployment.